pkg/status/metadata.yml

```
@@ -0,0 +1,3 @@
+status:
```

```suggestion
status:
  class: pkg
  stability:
    development: [ extension ]
```
We should include a few more details here just for completeness.